modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -15,6 +16,19 @@ def show
 
           render json: serialized.serializable_hash
         end
+
+        def upload
```

Comment:
instead of upload, could this be "#create"? If we really want the endpoint to be 'upload', we can always do a `as: 'upload`. Just to keep it RESTful. What do you think?

Reply:
Yes! I didn't context switch when i moved from the nod controller to the submission controller. I'll change it now.